features/302.ngsild.BigDataFlink.feature

```diff
@@ -0,0 +1,72 @@
+Feature: test tutorial 302 - Big Data Analysis (Flink)
+
+  url: https://ngsi-ld-tutorials.readthedocs.io/en/latest/big-data-flink.html
+
+  git-clone: https://github.com/FIWARE/tutorials.Big-Data-Flink.git
+  git-directory: /tmp/tutorials.Big-Data-Flink
+  shell-commands: git checkout NGSI-LD ; ./services create  ; ./services start
+  clean-shell-commands: ./services stop
+
+Background:
+    Given  I set the tutorial 302 LD - Big Data Flink
+
+    # COMPILING A JAR FILE FOR Flink --
+    Scenario: Compiling a Jar File For Flink
+       Given I prepare the script create-flink-jar.sh to be run
+       And   I set the environ variable WORKING_DIR to "cosmos-examples" under git
+       And   I set the environ variable OUTPUT_DIR to "cosmos-examples/target" under git
+       When  I run the script as in the tutorial page
+       Then  I expect the scripts shows a result of 0
+
+    Scenario Outline: Subscribing Logger to context changes
+    When I prepare a POST HTTP request for "<description>" to "http://localhost:1026/ngsi-ld/v1/subscriptions/"
+    And  I set header Content-Type to application/ld+json
```

Comment:
This might cause other things to break as in Header "Link" -- which value tend to be: '<http://context/ngsi-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"'

This has been accepted long time ago and it is extensively used in features. I suggest changing them all in a separate feature. And I suggest using single quotes instead of double quotes.

Reply:
NTC